doc/api/cli.md

```diff
@@ -773,6 +773,8 @@ greater than `4` (its current default value). For more information, see the
 [`Buffer`]: buffer.html#buffer_class_buffer
 [`SlowBuffer`]: buffer.html#buffer_class_slowbuffer
 [`process.setUncaughtExceptionCaptureCallback()`]: process.html#process_process_setuncaughtexceptioncapturecallback_fn
+[`tls.DEFAULT_MAX_VERSION`]: tls.html#tls_tls_default_max_version
+[`tls.DEFAULT_MIN_VERSION`]: tls.html#tls_tls_default_min_version
```

Comment:
I feel like these should be added to the other file?

Reply:
I back-ported this [commit](https://github.com/nodejs/node/pull/26821/files#diff-85b6573bc4afd5e06ed3b9d9b553635dR924) from cli.md to 10.x. However, if it is needed in other file (tls.md) I can certainly move/add it there. Do you have any suggestions on where to include it in the tls.md.

Reply:
On further analysis I realized as the cli options (--tls-max-vXX, --tls-min-vXX) are not supported in v10, there are no corresponding sections for command line option in this file. So these changes do not need to be back ported. I will revert these lines. Thanks.